gptcache/manager/vector_data/manager.py

```diff
@@ -203,6 +203,7 @@ def get(name, **kwargs):
             port = kwargs.get("port", "6379")
             user = kwargs.get("user")
             password = kwargs.get("password")
+            namespace = kwargs.get("namespace")
```

Comment:
need a default value, otherwise `f"{self.namespace}doc:"` will get `Nonedoc:`
so: kwargs.get("namespace", "") is better